src/components/common/Input/index.js

```diff
@@ -11,16 +11,20 @@ const Input = ({ label, value, onChangeText, error, active, touched, ...props })
   }, []);
 
   return (
-    <View>
-      {label && <Text>{label}</Text>}
-      <View>
+    <View style={styles.container}>
+      {label && <Text style={styles.label}>{label}</Text>}
```

Comment:
even if label is required it can be an empty string `""`, in this case I'll suggest you to safeguard this conditional render like
```suggestion
      {!!label && <Text style={styles.label}>{label}</Text>}
```

to avoid any unexpected crashes